### ISSUE TITLE
Verify our site counts are accurate for v1 sites

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#7481]
# Story

As a web team, we would like to verify our D7 site counts are accurate so that we can make sure all D7 sites get accounted for in our comms for SiteNow v1 standard, SiteNow v1 custom that can transition without development and SiteNow v1 custom that need custom development.

# Additional Info

We shouldn't send any comms until we have done this.

We discussed generating a list manually of site directories. We want to go to the site to confirm the site renders and that there is a databases & domain associated with the site on the 701-703 applications.

We want to make sure our manual list matches our site counts and if not we want to understand the discrepancies.

Eventually, we will want to differentiate the manually created list of sites into buckets - v1 standard, v1 custom, v1 custom that can be standard but that doesn't have to be in scope for this slice.

# Proposed Solution

**Comment by briand44** [comment]
Grooming conversation... Manually create list by looking at sites directory on prod server or using python. We want the number of records in the result to represent the number of sites. Any additional information we decide to capture from this should not be separate record but added to a specific column or notes. We want to understand any discrepancies from this list compared to site counts. Any changes to site counts or elsewhere could happen as a separate slice.

**Comment by pyrello** [comment]
I'm going to work on this.

**Comment by joewhitsitt** [comment]
Just my thoughts from EoD, not necessarily a scope increase is that if the final output of this info is a spreadsheet, it would be neat if it was compatible for an Excel merge with the existing `updated all` sheet in the d7 spreadsheet. It looked like `Site Url` and `Site` could be the unique key. The other columns could be merged or renamed to not conflictseparate column could be created to represent the results as a whole compared to what exists in the spreadshet. Don't want to lose the value of the existing spreadsheet and the data there (e.g. notes, etc.)

**Comment by joewhitsitt** [comment]
@pyrello take the time you need but the team was wondering at standup if you had anything capture so far so that one of use can try and keep pushing it forward in your absence without starting over.

**Comment by pyrello** [comment]
> @pyrello take the time you need but the team was wondering at standup if you had anything capture so far so that one of use can try and keep pushing it forward in your absence without starting over.

https://iowa.sharepoint.com/:x:/r/sites/CSI-Drupal2/Shared%20Documents/D7%20Site%20Count%20Verification.xlsx?d=w971457d3c0ca45d7a94491dff03a55f2&csf=1&web=1&e=0HccDY

**Comment by joewhitsitt** [comment]
Thanks for the link. I am going to try to fill this in a bit more

**Comment by joewhitsitt** [comment]
wowzers. this takes a bit. chugging along.

**Comment by joewhitsitt** [comment]
Not sure what else was planned for the spreadsheet, but following the pattern set by @pyrello I have finished the spreadsheet.

Added a v1 custom column based on my historic knowledge of the service, the themes and logging into a couple to verify.

>Eventually, we will want to differentiate the manually created list of sites into buckets - v1 standard, v1 custom, v1 custom that can be standard but that doesn't have to be in scope for this slice.

Sounds like that is being carved off in https://github.com/uiowa/uiowa/issues/7506

---

By my count, just looking at discontinued/moved/replaced we could remove 51 websites from SiteNow v1 standard/custom.
